src/rootSaga.js

```diff
@@ -0,0 +1,7 @@
+import { facebookSaga, googleSaga } from "./sagas/LoginSagas";
+import { spawn } from "redux-saga/effects";
+
+export default function* rootSaga() {
+  yield spawn(facebookSaga);
```

Comment:
I've never used **spawn** - could you tell me why did you choose it?

Reply:
Spawn will create detached task - it will remain independent from it's parent and will act like top-level task. Parent won't wait for task to terminate, errors and cancellations are also detached - which means error in one task won't cancel whole saga.
I saw this approach and I guessed it's ok, what's your opinion tho?